apps/bleskomat-server/docker-compose.yml

```
@@ -0,0 +1,28 @@
+version: "3.7"
+
+services:
+  web:
+    image: bleskomat/bleskomat-server:1.2.0@sha256:1566bee49252c2aa431094f07fdb6ad1cb71eb0e916bfe7484f9d54e75b38bf1
```

This is pinning to the hash of the arm64 image. We need to pin to the multiarch manifest so the app works on both arm64 and amd64 Umbrel installs.

```suggestion
    image: bleskomat/bleskomat-server:1.2.0@sha256:278703d6ada3f038158bf3799dcefaa4d38cffff25e63e20f0bced23ef1ae5ca
```

For future reference you can get the multiarch manifest hash from `docker pull`:

```
$ docker pull bleskomat/bleskomat-server:1.2.0
1.2.0: Pulling from bleskomat/bleskomat-server
Digest: sha256:278703d6ada3f038158bf3799dcefaa4d38cffff25e63e20f0bced23ef1ae5ca
Status: Image is up to date for bleskomat/bleskomat-server:1.2.0
docker.io/bleskomat/bleskomat-server:1.2.0
```

---

Thanks for the tip. I'll be sure to use the multiarch sha256 hash in the future.